website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -775,6 +817,94 @@ spec:
 ```
 {{% /alert %}}
 
+### AL2023
+
+* Your UserData may be in one of three formats: a [MIME multi part archive](https://cloudinit.readthedocs.io/en/latest/topics/format.html#mime-multi-part-archive), a NodeConfig YAML / JSON string, or a shell script.
+* Karpenter will transform your custom UserData as a MIME part, if necessary, and then merge a generated MIME part to the **beginning** of the the generated MIME multi part archive. Since `nodeadm`, the bootstrap for the AL2023 EKS optimized AMI, merges NodeConfig objects from top to bottom any fields specified in the custom UserData will override Karpenter generated fields. Take caution when overriding these fields, misconfiguration can result in Nodes being unable to connect to your cluster.
```

Comment:
```suggestion
* Karpenter will transform your custom UserData as a MIME part, if necessary, and then merge a generated MIME part to the **beginning** of the the generated MIME multi part archive. Since the AL2023 bootstrap logic merges NodeConfig objects from top to bottom, any fields specified in the custom UserData will override Karpenter-generated fields. Take caution when overriding these fields, misconfiguration can result in Nodes being unable to connect to your cluster.
```

I think you can reduce the wordiness this way. Also, users probably don't care that the internal script is called `nodeadm`